examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -37,8 +37,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    from recommenders.datasets.amazon_reviews import _reviews_preprocessing
</pre><p>minor comment, in other parts of the code, if we want to use a function in a notebook, we add it as a public, so would be <code>reviews_preprocessing</code>  instead of <code>_reviews_preprocessing</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I am re-using one of the existing functions. This was not directly called from anywhere earlier but <code>data_processing().</code></p>

Reply:
<p>I also think it is better to hide the private method behind a public one i.e. you could move the code in which you use this from this notebook to a function in amazon_reviews.py. &nbsp;</p>